ntbs-service/Pages/ServiceDirectory/Region.cshtml

```diff
@@ -46,8 +47,12 @@
                         </span>
                     </h3>
                 </div>
-                <div id="accordion-content-@tbService.Code" class="govuk-accordion__section-content" 
-                    aria-labelledby="accordion-heading-@tbService.Code">
+                <div id="accordion-content-@tbService.Code" class="govuk-accordion__section-content"
+                     aria-labelledby="accordion-heading-@tbService.Code">
+                    @if (hospitals.Any())
+                    {
+                        <p>Hospitals: @string.Join(", ", hospitals)</p>
```

Comment:
Very minor thing, but it's cleaner to separate your logic from your view. So here, I would have a variable just below where you've defined `hospitals` called something like `hospitalsString` and set it to `string.Join(", ", hospitals)`